_data/retail.yml

```diff
@@ -28,6 +28,13 @@ websites:
       bch: No
       twitter: AliExpress_EN
 
+    - name: Ali Lightning
```

Comment:
Travis hasnt run fully but willing to bet job 2 on it will indicate that this is not alphabetically correct as a space comes before E so this belongs before AliExpress

Reply:
Seems like it passed, is the test or the ordering wrong?

Reply:
Re-ordered so Ali Lightning is before AliExpress